packages/xo-server/src/_handleBackupLog.mjs

```diff
@@ -1,3 +1,30 @@
+import humanFormat from 'human-format'
+import ms from 'ms'
+
+const vmBackupInfo = new Map()
+
+function sendToNagios(app, jobName, vmBackupInfo) {
+  try {
+    vmBackupInfo.size = humanFormat(vmBackupInfo.size, { scale: 'binary', unit: 'B' })
+    vmBackupInfo.duration = ms(vmBackupInfo.end - vmBackupInfo.start)
+
+    delete vmBackupInfo.type
+    delete vmBackupInfo.start
+    delete vmBackupInfo.end
+
+    app.sendPassiveCheck(
+      {
+        message: JSON.stringify(vmBackupInfo),
+        status: 0,
+      },
+      app.getObject(vmBackupInfo.vmId).name_label.toLowerCase(),
```

Comment:
Why `toLowerCase`?

Reply:
because in my test case the host name on XO starts with a capital letter while the host name on Nagios config is entirely in lower case.

But you're right we should assume that the user uses the same format for both.